doc/input-output-reference/src/energyplus-economics/utilitycost-ratchet.tex

```diff
@@ -102,8 +102,8 @@ \subsection{Inputs}\label{inputs-071}
 
 \paragraph{Field: Multiplier Value or Variable Name}\label{field-multiplier-value-or-variable-name}
 
-Often the ratchet has a clause such as ``the current month demand or 90\% of the summer month demand''. For this case a value of 0.9 would be entered here as the multiplier. This value may be left blank if no multiplier is needed, and a value of zero (same as blank) is used as the default.
+Often the ratchet has a clause such as ``the current month demand or 90\% of the summer month demand''. For this case a value of 0.9 would be entered here as the multiplier. This value may be left blank if no multiplier is needed, and a value of one will be used as a default.
 
 \paragraph{Field: Offset Value or Variable Name}\label{field-offset-value-or-variable-name}
 
-A less common strategy is to say that the ratchet must be all demand greater than a value, in this case an offset that is added to the demand may be entered here. If entered, it is common for the offset value to be negative representing that the demand be reduced.~ If no value is entered it is assumed to be zero and not affect the ratchet.
\ No newline at end of file
+A less common strategy is to say that the ratchet must be all demand greater than a value, in this case an offset that is added to the demand may be entered here. If entered, it is common for the offset value to be negative representing that the demand be reduced.~ If no value is entered it, is assumed to be zero and not affect the ratchet.
```

Comment:
There are still some duplicate label warnings in the Inverter output variables section. Since those need to be fixed, I'll mention the misplaced comma here: 
If no value is entered it, is assumed to be zero